Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_expeditions_explorers.yml

```diff
@@ -345,6 +345,10 @@
       200: Dead
   - type: Stamina
     critThreshold: 600
+  - type: SlowOnDamage
+    speedModifierThresholds:
+      120: 0.7
+      160: 0.5
```

Comment:
Leaving this as a question, don't know the answer right now - do the other explorers inherit this from some base entity?  Is that why there aren't entries for gangers and mercs?  Default health, default SlowOnDamage values?

Reply:
Got a response - yes, they should.